docs/install/upgrade.md

```diff
@@ -8,6 +8,9 @@ Upgrading Known is designed to be very simple.
 
 That's it!
 
+!!! note "Github users"
+    If you're using a direct checkout from Github (rather than using one of the prepared packages), you need to update any dependencies using ```composer update```
```

Comment:
```suggestion
    If you're using a direct checkout from GitHub (rather than using one of the prepared packages), you need to update any dependencies using ```composer update```
```